app/Models/Category.php

```diff
@@ -45,7 +45,7 @@ public function nbNotRead() {
 		return $this->nbNotRead;
 	}
 	public function feeds() {
-		if ($this->feeds === null) {
+		if (null === $this->feeds) {
```

Comment:
Any advantage of writing the comparison in reverse?

Reply:
On this one not really, but when you're working with variables, you're sure you are not assigning a variable by mistake. For instance:

    // This is always true
    if ($var = 'hello')...

    // This fails
    if ('hello' = $var)...

I found that it's a good habit to have tests in reverse. My 2¢